index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine|Inconsolata|Droid+Sans|Poppins">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700:900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+    <title>Porfolio</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header id="header"> 
+            <div class="logo blue-text"><a href="#">Joeveloper</a></div>
+            <nav class="nav-bar">
+                <ul id="navlink-ul">
+                    <li><a class="nav-links black-text" href="">Portfolio</a></li>
+                    <li><a class="nav-links black-text" href="">About</a></li>
+                    <li><a class="nav-links black-text" href="">Contact</a></li>
+                    <li><a class="nav-links black-text" href=""><i class="fas fa-envelop"></i></a></li>
+                </ul>
+                <div class="burger-parents">
+                    <div class="hamburger"></div>
+                    <div class="hamburger"></div>
+                    <div class="hamburger"></div>
+                </div>
+                
+        </nav>
+    </header>
+
+    <section id="portfolio-main">
+        <div id="div">
+            <div class="main-intro">
+                <div id="maintext-body">
+                    <h1 class="intro-heading .black-text">I'm Joeveloper</h1>
+                    <h1 class="intro-heading">Glad to see you!</h1>
+                    <p id="intro-para" class="black-text">I'm a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</p>
+                </div>
+            </div>
+            <div id="social-media">
+                <p id="social-para" class="social-blue">LET'S CONNECT</p>
+                    <ul>
+                        <li><a target="_blank" class="social-tag" href="https://www.github.com/joeveoper"><img src="/images/twitter.png"></a></li>
+                        <li><a target="_blank" class="social-tag" href="https://www.linkedin.com/in/yinka-olatunji-83aa36b5/"><img src="/images/vector.png"></a></li>
+                        <li><a target="_blank" class="social-tag" href="https://www.twitter.com/joeveoper"><img src="/images/medium.png"></a></li>
+                        <li><a target="_blank" class="social-tag" href="https://www.medium.com/"><img src="/images/github.png"></a></li>
+                        <li><a target="_blank" class="social-tag" href="https://www.angellist.com/"><img src="/images/angellist.png"></a></li>
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better the icons visible on your webpage to replicate the Figma design and for the best user experience 👍 👍 

![image](https://user-images.githubusercontent.com/102757126/219647569-8ca09dfc-8323-408d-ac81-0d6999a6b834.png)

Reply:
![Screenshot 2023-02-17 135857](https://user-images.githubusercontent.com/36210790/219659465-f4daa4df-ccb4-4f8f-b25e-04a7444aacbd.png)

the images are visible from here ....